packages/next-server/server/config.ts

```diff
@@ -39,6 +39,8 @@ const defaultConfig: { [key: string]: any } = {
     documentMiddleware: false,
     publicDirectory: false,
   },
+  serverRuntimeConfig: {},
+  publicRuntimeConfig: {},
```

Comment:
This was not set intentionally as it will propagate into __NEXT_DATA__ when set

Reply:
It's updated to not propagate to `__NEXT_DATA__` if it's an empty object 👍